csfieldguide/package.json

```diff
@@ -13,11 +13,13 @@
     "sticky-state": "2.4.1",
     "iframe-resizer": "3.6.3",
     "csfg-module-fsa": "file:./static/js/modules/fsa/",
+    "csfg-interactive-algorithm-timer": "file:./static/interactives/algorithm-timer/",
     "csfg-interactive-big-number-calculator": "file:./static/interactives/big-number-calculator/",
     "csfg-interactive-bin-packing": "file:./static/interactives/bin-packing/",
```

Comment:
Dragging a box in bin-packing will also highlight text as though you were doing a click and drag to select a group of stuff

Reply:
Perhaps also address #786 if you think it's needed

Reply:
Text highlighting error is fixed. Not sure what to add for #786 happy to hear suggestions